### PR TITLE
Disable kroki links by default.

### PR DESCRIPTION
**Pull request by windiana42**
# Checklist

- [x] Added a `docs/source/changelog.md` entry

### CODE REVIEW

**Comment by windiana42** [comment]
@DominikZuercherQC FYI

File: README.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ def main():
         cfg = create_basic_pipedag_config(
             f"duckdb:///{temp_dir}/db.duckdb",
             disable_stage_locking=True,  # This is special for duckdb
+            # Attention: If uncommented, task names might be sent to the following URL.
```
Comment (NicolasMuellerQC) [inline README.md:199]:
```suggestion
            # Attention: If uncommented, stage and task names might be sent to the following URL.
```

File: docs/package/README.md [hunk 2]
```diff
@@ -87,6 +87,10 @@ def main():
         cfg = create_basic_pipedag_config(
             f"duckdb:///{temp_dir}/db.duckdb",
             disable_stage_locking=True,  # This is special for duckdb
+            # Attention: If uncommented, task names might be sent to the following URL.
```
Comment (NicolasMuellerQC) [inline docs/package/README.md:90]:
```suggestion
            # Attention: If uncommented, stage and task names might be sent to the following URL.
```

File: docs/source/database_testing.md [hunk 3]
```diff
@@ -111,6 +111,12 @@ instances:
       - "sqlalchemy.sql.expression.TextClause"
       - "sqlalchemy.sql.expression.Selectable"
 
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline docs/source/database_testing.md:114]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: docs/source/reference/config.md [hunk 4]
```diff
@@ -484,6 +494,12 @@ technical_setups:
     # abort as fast a possible on task failure and print most readable stack trace
     fail_fast: true
 
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline docs/source/reference/config.md:497]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: example/run_pipeline.py [hunk 5]
```diff
@@ -66,6 +66,10 @@ def main():
         cfg = create_basic_pipedag_config(
             f"duckdb:///{temp_dir}/db.duckdb",
             disable_stage_locking=True,  # This is special for duckdb
+            # Attention: If uncommented, task names might be sent to the following URL.
```
Comment (NicolasMuellerQC) [inline example/run_pipeline.py:69]:
```suggestion
            # Attention: If uncommented, stage and task names might be sent to the following URL.
```

File: example_postgres/pipedag.yaml [hunk 6]
```diff
@@ -8,6 +8,13 @@ instances:
 
     fail_fast: true
     instance_id: pipedag_default
+
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline example_postgres/pipedag.yaml:12]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: pipedag.yaml [hunk 7]
```diff
@@ -40,6 +40,12 @@ instances:
       - "sqlalchemy.sql.expression.TextClause"
       - "sqlalchemy.sql.expression.Selectable"
 
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline pipedag.yaml:43]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: tests/test_flows/complex_config_flows/pipedag_anchor.yaml [hunk 8]
```diff
@@ -31,6 +31,12 @@ _technical_setups:
     # abort as fast a possible on task failure and print most readable stack trace
     fail_fast: true
 
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline tests/test_flows/complex_config_flows/pipedag_anchor.yaml:34]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: tests/test_flows/complex_config_flows/pipedag_complex.yaml [hunk 9]
```diff
@@ -34,6 +34,12 @@ technical_setups:
     # abort as fast a possible on task failure and print most readable stack trace
     fail_fast: true
 
+    # Attention: For disable_kroki: false, task names might be sent to the kroki_url.
```
Comment (NicolasMuellerQC) [inline tests/test_flows/complex_config_flows/pipedag_complex.yaml:37]:
```suggestion
    # Attention: For disable_kroki: false, stage and task names might be sent to the kroki_url.
```

File: docs/source/reference/config.md [hunk 10]
```diff
@@ -217,6 +217,16 @@ network_interface
 
   (default: `127.0.0.1`)
 
+disable_kroki
+: If set to `True`, [Kroki](https://kroki.io) URL will not be displayed at end of executing flow. Kroki URLs are a really nice way 
+of visualizing the flow execution without the need to install graphviz and to worry about how to display the result. 
+The graph is sent to the kroki_url only once you click the link. The whole graph is encoded in the URL. However, since
+there is the risk to expose sensitive information by sending task names to a public server, it is disabled by default.
```
Comment (windiana42) [inline docs/source/reference/config.md:224]:
```suggestion
there is the risk to expose sensitive information by sending stage and task names to a public server, it is disabled by default.
```

**Pull request merged**